mtp_noms_ops/apps/security/views/check.py

```diff
@@ -28,6 +30,44 @@ class CreditsHistoryListView(SecurityView):
     form_class = CreditsHistoryListForm
 
 
+class CheckAssignView(BaseFormView):
+    """
+    Modify assignment of check
+    """
+    form_class = AssignCheckToUserForm
+    id_kwarg_name = 'check_id'
+    object_context_key = 'check'
```

Comment:
this isn't referenced anywhere, is it needed?

Reply:
Apparently not.